src/screens/RideModeStack/RideModeHome.jsx

```diff
@@ -2,18 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Button, Container, Text } from 'native-base';
 
-const RideModeHome = ({ navigation }) => (
-  <Container>
-    <Button onPress={() => navigation.push('DropOffSubmit')}>
-      <Text>To bike drop off submit</Text>
-    </Button>
-  </Container>
-);
+const RideModeHome = ({ navigation, route }) => {
+  const bike = route.params.bike;
+  return (
+    <Container>
+      <Button onPress={() => navigation.push('DropOffSubmit')}>
+        <Text>To bike drop off submit</Text>
+      </Button>
+    </Container>
+  );
+};
 
 RideModeHome.propTypes = {
   navigation: PropTypes.shape({
     push: PropTypes.func.isRequired,
   }).isRequired,
+  route: PropTypes.isRequired,
```

Comment:
You're missing any details about what the route is, (func, obj, number, etc.). I'm surprised the linter let you get away with this lol.